### PR TITLE
zh-CN: translate remote playback API

### PR DESCRIPTION
**Pull request by xuanzhi33**
### Description

zh-CN: translate remote playback API

### CODE REVIEW

File: files/zh-cn/web/api/remote_playback_api/index.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+## 概念和用法
+
+远程播放设备是指电视、投影仪或扬声器等已连接的设备。此 API 考虑了通过 HDMI 或 DVI 连接的有线设备，以及 Chromecast 或 AirPlay 等无线设备。
+
+该 API 使具有媒体元素（如视频或音频文件）的页面能够在连接的远程设备上启用和控制该媒体的播放。例如，在连接的电视上播放视频。
+
+> **备注：** Safari for iOS 有一些在 AirPlay 上启用远程播放的 API。这些细节可以在 [Safari 9.0 发行说明](https://developer.apple.com/library/archive/releasenotes/General/WhatsNewInSafari/Articles/Safari_9_0.html#//apple_ref/doc/uid/TP40014305-CH9-SW16) 中找到。
+>
+> Android 版本的 Firefox 和 Chrome 也包含一些远程播放功能。如果本地网络中有可用的投放（Cast）设备，这些设备将显示一个投放（Cast）按钮。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/remote_playback_api/index.md:20]:
```suggestion
> **备注：** Safari for iOS 有一些在 AirPlay 上启用远程播放的 API。这些细节可以在 [Safari 9.0 发行说明](https://developer.apple.com/library/archive/releasenotes/General/WhatsNewInSafari/Articles/Safari_9_0.html#//apple_ref/doc/uid/TP40014305-CH9-SW16)中找到。Android 版本的 Firefox 和 Chrome 也包含一些远程播放功能。如果本地网络中有可用的投放（Cast）设备，这些设备将显示一个投放（Cast）按钮。
```

File: files/zh-cn/web/api/remote_playback_api/index.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+**远程播放 API** 扩展了 {{domxref("HTMLMediaElement")}} 来控制远程设备上播放的媒体。
+
+## 概念和用法
+
+远程播放设备是指电视、投影仪或扬声器等已连接的设备。此 API 考虑了通过 HDMI 或 DVI 连接的有线设备，以及 Chromecast 或 AirPlay 等无线设备。
+
+该 API 使具有媒体元素（如视频或音频文件）的页面能够在连接的远程设备上启用和控制该媒体的播放。例如，在连接的电视上播放视频。
+
+> **备注：** Safari for iOS 有一些在 AirPlay 上启用远程播放的 API。这些细节可以在 [Safari 9.0 发行说明](https://developer.apple.com/library/archive/releasenotes/General/WhatsNewInSafari/Articles/Safari_9_0.html#//apple_ref/doc/uid/TP40014305-CH9-SW16)中找到。Android 版本的 Firefox 和 Chrome 也包含一些远程播放功能。如果本地网络中有可用的投放（Cast）设备，这些设备将显示一个投放（Cast）按钮。
```
Comment (yin1999) [inline files/zh-cn/web/api/remote_playback_api/index.md:18]:
```suggestion
> **备注：** Safari for iOS 有一些在 AirPlay 上启用远程播放的 API。这些细节可以在 [Safari 9.0 发行说明](https://developer.apple.com/library/archive/releasenotes/General/WhatsNewInSafari/Articles/Safari_9_0.html#//apple_ref/doc/uid/TP40014305-CH9-SW16)中找到。Android 版本的 Firefox 和 Chrome 也包含一些远程播放功能。如果本地网络中有可用的投放（Cast）设备，这些设备将显示投放按钮。
```

File: files/zh-cn/web/api/remote_playback_api/index.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+## 接口
+
+- {{domxref("RemotePlayback")}}
+  - : 允许页面检测远程播放设备的可用性，并连接并控制这些设备上的播放。
```
Comment (yin1999) [inline files/zh-cn/web/api/remote_playback_api/index.md:23]:
```suggestion
  - : 允许页面检测远程播放设备的可用性，然后连接和控制这些设备上的播放。
```

**Review by yin1999 — APPROVED** [review]
感谢 @xuanzhi33、@skyclouds2001

**Pull request merged**